### DISCUSSION TITLE
Welcome to GIP Discussions!

### DISCUSSION THREAD

**Discussion by giovannibussotti** [giovannibussotti/GIP#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cnrahul** [comment]
Dear Giovannibussotti,

I am a Post Doctoral fellow with research interest in malaria. I am interested in learning this GIP tutorial. I am new  to coding and computational biology. However, I have an intent to learn and very excited to know about GIP pipeline. I only have a basic i5 supported laptop with ubuntu installed will this be  computationally sufficient to do the tasks.

**Reply by giovannibussotti** [reply]
Hi Rahul,
thank you for your interest in GIP. It really depends on the library size of your dataset and the number of samples. The most CPU and memory expensive step is the read alignment performed with BWA. Analyzing ~200 Plasmodium vivax samples described in our paper (https://www.biorxiv.org/content/10.1101/2021.06.15.448580v2.full) the memory peak we measured was ~34 Gb. 

<img width="1161" alt="Screenshot 2021-11-02 at 14 44 29" src="https://user-images.githubusercontent.com/12170377/139859012-2b9480ec-457e-4f36-a5ee-fc2d66c5d8cf.png">
.
